src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -4030,6 +4030,7 @@ public override void Add(View child)
                     };
                     ChildAdded(this, e);
                 }
+                BindableObject.SetInheritedBindingContext(child, this?.BindingContext);
```

Comment:
please add this "BindableObject.SetInheritedBindingContext(child, this?.BindingContext);" into Layer. 
=> Layer.Add(). I tested and it is needed.

Reply:
Got it.